# Copyright 2024 Shadow Robot Company Ltd.
#
# This program is free software: you can redistribute it and/or modify it
# under the terms of the GNU General Public License as published by the Free
# Software Foundation version 2 of the License.
#
# This program is distributed in the hope that it will be useful, but WITHOUT
# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
# more details.
#
# You should have received a copy of the GNU General Public License along
# with this program. If not, see <http://www.gnu.org/licenses/>.

---
sr_tf_live_republisher:
  output_topic: '/tf_filtered_record'
  frequency: 100
  parent_to_child_frames:
    rh_palm: rh_fftip
    rh_palm: rh_mftip
    rh_palm: rh_rftip
    rh_palm: rh_lftip
    rh_palm: rh_thtip